text/0146-deflationary-fee-proposal.md

```diff
@@ -0,0 +1,47 @@
+# RFC-0146: Deflationary Transaction Fee Model for Relay- and System Chains
```

Comment:
Probably a nit since it's consistent and clear from context, but "System Chains" != "System Parachains", usually "System Chains" for a given network would include the Relay Chain.

Reply:
Thanks for the insight, that would mean "Deflationary Transaction Fee Model for Polkadot's System Chains" is a more precise title?

Reply:
Okay, since I assume that I'm not the only one that did not know this nuance, I'll update the RFC to talk about the "Relay Chain and its System Parachains".